implementation/src/main/java/io/smallrye/jwt/config/JWTAuthContextInfoProvider.java

```diff
@@ -226,6 +246,20 @@ private static JWTAuthContextInfoProvider create(String publicKey, String public
     @ConfigProperty(name = "smallrye.jwt.verify.aud")
     Optional<Set<String>> expectedAudience;
 
+    /**
+     * The JWT signature verification algorithm to use.
+     */
+    @Inject
+    @ConfigProperty(name = "io.smallrye.jwt.verify.algorithm", defaultValue = "RS256")
+    String signatureAlgorithm;
+
+    /**
+     * The HMAC secret if HMAC is used.
+     */
+    @Inject
+    @ConfigProperty(name = "io.smallrye.jwt.verify.hmac-secret")
```

Comment:
As suggested at #126, IMHO it should be `secret-key-location`, I'm not also that 'hmac' should be in the name, in case we decide to support some other secret key signature algorithm.

Reply:
The other thing a `.verify.` fragment might be redundant, because for the JWT signing utilities it would also be a signing secret key loation